app/models/company.rb

```diff
@@ -21,11 +21,19 @@ def get_info
         compensationandbenefits_rating: result["compensationAndBenefitsRating"].to_f,
         careeropportunities_rating: result["careerOpportunitiesRating"].to_f,
         worklifebalance_rating: result["workLifeBalanceRating"].to_f,
-        recommendtofriends_rating: result["recommendToFriendRating"].to_f,
+        recommendtofriends_rating: result["recommendToFriendRating"].to_f
+        )
+    end
+
+    if result && result["ceo"]
```

Comment:
It looks like the ceo key is usually available, but not necessarily the ["ceo"]["image"].